.changeset/smooth-carpets-matter.md

```diff
@@ -1,18 +1,18 @@
 ---
-'babel-plugin-emotion': major
-'@emotion/core': major
+'@emotion/babel-plugin': major
+'@emotion/react': major
 '@emotion/css': major
 ---
 
-Removed `@emotion/css` - it's main purpose was to allow `css` to be a Babel macro, but since babel-plugin-macros allows us to keep imports nowadays this is no longer needed. `@emotion/core/macro` has been added to account for this use case and appropriate changes has been made to `babel-plugin-emotion` to facilitate those changes.
+Removed `@emotion/css` - it's main purpose was to allow `css` to be a Babel macro, but since babel-plugin-macros allows us to keep imports nowadays this is no longer needed. `@emotion/react/macro` has been added to account for this use case and appropriate changes has been made to `@emotion/babel-plugin` to facilitate those changes.
 
-If you have used `@emotion/css` directly (it was always reexported from `@emotion/core`) or you have been using its macro then you should update your code like this:
+If you have used `@emotion/css` directly (it was always reexported from `@emotion/react`) or you have been using its macro then you should update your code like this:
```

Comment:
We should review all changesets & changelogs carefully after merging this in. Some changesets should still refer to old names, like for example in here - it should still stay as "(it was always reexported from `@emotion/core`".

And probably best to keep copies of existing changelogs in old and new packages?

Reply:
bump - we should mentioned here that it was always reexported from `@emotion/core`, optionally we can mention that it has been renamed now to `@emotion/react`